authors/apps/articles/views.py

```diff
@@ -302,15 +296,39 @@ def delete(self, request, slug, vote_type):
         A method to delete a vote for a certain article
         """
         article = RetrieveUpdateArticleAPIView().retrieve_article(slug)
-        if article.votes.delete(request.user.id):
+
+        def if_vote_exist(vote):
+            if article.votes.exists(request.user.id, vote):
+                return True
+            return False
+
+        def process_vote_type(vote_type):
+            vote_message = None
+            voted = False
+            if vote_type == "like" and if_vote_exist(0):
+                vote_message = ["Like delete successfully"]
+                voted = True
+            elif vote_type == "dislike" and if_vote_exist(1):
+                vote_message = ["Dislike delete successfully"]
+                voted = True
+            response = Response({
+                "message": vote_message
```

Comment:
Please return the entire article object just like when liking or disliking an article.